src/interfaces/IMetaMorpho.sol

```diff
@@ -7,8 +7,8 @@ import {IERC4626} from "@openzeppelin/interfaces/IERC4626.sol";
 struct MarketConfig {
     /// @notice The maximum amount of assets that can be allocated to the market.
     uint192 cap;
-    /// @notice The rank of the market in the withdraw queue.
-    uint64 withdrawRank;
+    /// @notice Whether the market is in the withdraw queue.
+    bool enabled;
```

Comment:
We may also want to switch the order to have the masked data part be the less used
1. the less used in the code is clearly `enabled`
2. but currently, is it `enabled` that needs to be masked to be read or is it `cap`?

Reply:
To get any value in a slot you shift it at the beginning of the word, and mask it afterwards. You can optimize this by:
1. not shifting when the value is already the beginning of the slot
2. not mask it when the value is already at the end of the slot

It seems like solidity does not do optimization 2, see [this PR](https://github.com/morpho-labs/morpho-sandbox/pull/16). So in theory it would cost less to put the most used at the beginning of the struct/slot.

But I think that this is such a small optimization that we should not bother with it at all

Reply:
Thanks for investigating!